src/Http/Http/src/Features/FormOptions.cs

```diff
@@ -16,83 +16,124 @@ public class FormOptions
         /// <summary>
         /// Default value for <see cref="MemoryBufferThreshold"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 64KB.
+        /// </value>
         public const int DefaultMemoryBufferThreshold = 1024 * 64;
 
         /// <summary>
         /// Default value for <see cref="BufferBodyLengthLimit"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>134,217,728 bytes‬</c>, which is 128MB.
+        /// </value>
         public const int DefaultBufferBodyLengthLimit = 1024 * 1024 * 128;
 
         /// <summary>
         /// Default value for <see cref="MultipartBoundaryLengthLimit"/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>128 bytes‬</c>.
+        /// </value>
         public const int DefaultMultipartBoundaryLengthLimit = 128;
 
         /// <summary>
         /// Default value for <see cref="MultipartBodyLengthLimit "/>.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>134,217,728 bytes‬</c>, which is approximately 1.07GB.
+        /// </value>
         public const long DefaultMultipartBodyLengthLimit = 1024 * 1024 * 128;
 
         /// <summary>
         /// Enables full request body buffering. Use this if multiple components need to read the raw stream.
-        /// The default value is false.
         /// </summary>
+        /// <value>
+        /// Defaults to <see langword="false" />.
+        /// </value>
         public bool BufferBody { get; set; } = false;
 
         /// <summary>
         /// If <see cref="BufferBody"/> is enabled, this many bytes of the body will be buffered in memory.
         /// If this threshold is exceeded then the buffer will be moved to a temp file on disk instead.
         /// This also applies when buffering individual multipart section bodies.
         /// </summary>
+        /// <value>
+        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
+        /// </value>
```

Comment:
Still have "Defaults to" with the settable properties.

```suggestion
        /// <summary>
        /// If <see cref="BufferBody"/> is enabled, this many bytes of the body will be buffered in memory.
        /// If this threshold is exceeded then the buffer will be moved to a temp file on disk instead.
        /// This also applies when buffering individual multipart section bodies.
        /// Defaults to <c>65,536 bytes‬</c>, which is approximately 0.52MB.
        /// </summary>
```